cmd/monaco/generate/deletefile/deletefile.go

```diff
@@ -185,6 +189,20 @@ func skipping(ttype string, included, excluded map[string]struct{}) bool {
 	return false
 }
 
+func toMapKey(d persistence.DeleteEntry) string {
+	return fmt.Sprintf("%s/%s/%s/%s/%s/%s", d.Project, d.Type, d.ConfigId, d.ConfigName, d.Scope, sortedKeyValues(d.CustomValues, "/"))
```

Comment:
Would it matter if the same key was generated for different entries? Here it is certainly possible, even if unlikely.